help/fluid.labelset-help.pd

```diff
@@ -213,28 +165,62 @@ labels)., f 63;
 #X connect 12 0 14 0;
 #X connect 14 0 10 0;
 #X restore 477 658 pd other_messages;
-#X obj 36 596 bng 15 250 50 0 empty empty empty 17 7 0 10 #fcfcfc #000000
-#000000;
-#X obj 57 595 cnv 15 15 15 empty empty empty 20 12 0 14 #c6ffc7 #404040
-0;
-#X obj 37 677 cnv 15 15 15 empty empty empty 20 12 0 14 #c6ffc7 #404040
-0;
+#X obj 36 596 bng 15 250 50 0 empty empty empty 17 7 0 10 #fcfcfc #000000 #000000;
+#X obj 57 595 cnv 15 15 15 empty empty empty 20 12 0 14 #c6ffc7 #404040 0;
+#X obj 37 677 cnv 15 15 15 empty empty empty 20 12 0 14 #c6ffc7 #404040 0;
 #X text 58 594 6) store dataset into a text object;
 #X text 38 675 7) double-click 'text' to see data;
 #X obj 36 507 fluid.labelset help.labelset;
-#X obj 36 621 fluid.ls2text help.labelset help.labelset.text, f 31
-;
+#X obj 36 621 fluid.ls2text help.labelset help.labelset.text, f 31;
 #X obj 36 700 text define help.labelset.text;
 #N canvas 232 50 708 166 using_text 0;
-#X obj 17 13 cnv 15 660 42 empty empty fluid.dataset 1 22 0 36 #4ba3fb
-#ffffff 0;
-#X text 17 79 As demonstrated in the previous patchers \, the helper
-[fluid.ls2text] retrieves the identifier->label pairs in a provided
-text object., f 79;
-#X restore 477 680 pd using_text;
+#X obj 17 13 cnv 15 660 42 empty empty fluid.dataset 1 22 0 36 #4ba3fb #ffffff 0;
+#X text 17 79 As demonstrated in the previous patchers \, the helper [fluid.ls2text] retrieves the identifier->label pairs in a provided text object., f 79;
+#X restore 477 702 pd using_text;
 #X obj 230 537 print right;
 #X obj 36 539 print left;
-#X obj 477 703 note_on_automatic_datasets;
+#X obj 477 725 note_on_automatic_datasets;
+#N canvas 131 150 695 624 merging 0;
+#X obj 17 13 cnv 15 660 42 empty empty fluid.labelset 1 22 0 36 #4ba3fb #ffffff 0;
+#X text 17 79 You can merge the points in one dataset with another. If the dataset you call merge on already has a key existing in the dataset you want to merge \, the caller's will be preserved. Any keys not found in the dataset to merge will be added to the caller., f 84;
```

Comment:
This subpatch says 'dataset' in a number of comments where it should say 'labelset'